### ISSUE TITLE
API? czy masz jakies API

### ISSUE THREAD

**Issue by shivetay** [lukasz6678/school#7]

**Comment by shivetay** [comment]
https://www.npmjs.com/package/json-server

**Comment by lukasz6678** [comment]
Nie, póki co wgl nie zajmowałem się backendem w projekcie. Jedyne co, to podczas pierwszego kursu (ZTM Andrea Nagoie'a) dotknęliśmy nieco Express.js. Spróbuję to doczytać. Dziękuję!!
